msg/RoadPolyline.msg

```diff
@@ -0,0 +1 @@
+sensr_msgs/RoadPolylinePoint[] points
```

Comment:
seems like you can use `geometry_msgs/Vector3.msg` instead

Reply:
http://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Vector3.html

Reply:
Yes, thanks!